spanner/api/Spanner.Samples/InsertSampleDataAsync.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_insert_data]
```

Comment:
This is the context for the sample in the [Getting Started Guide](https://cloud.google.com/spanner/docs/getting-started/csharp#write-data-with-mutations). I think once again, these are not stand-alone samples, they are used for a broader workflow of getting a bunch of tables setup so that you can do more complex operations later in the guide.

Reply:
See previous comment. It's fine as it is.